servers/dsub/jobs/controllers/utils/generator.py

```diff
@@ -84,23 +83,19 @@ def generate_jobs(provider, query, create_time_max=None, offset_id=None):
         yield j
 
 
-def generate_jobs_count(provider,
-                        project_id,
-                        window_min,
-                        window_max=datetime.datetime.now(tz=tzlocal())):
+def generate_jobs_by_window(provider, project_id, window_min, window_max=None):
     """Get the generator of jobs for aggregation.
 
```

Comment:
Please specify that this is returning jobs that ended within the given window. And that it includes still running jobs if no window_max is specified.

Reply:
I planed to have running job aggregation shown on the dashboard since the mock includes one. Do we need it?